Compilation failure on docker-dev-run

I'm trying to run `make docker-dev-run` on macOS Catalina 10.15.6

First I'm seeing this message (Which I'm not sure relevant to the case):

    make: python3.8: Command not found

And then:
```
react-server    | ./src/util/i18n.js
react-server    | Module not found: Can't resolve '../locales/en/messages' in '/newdle/client/src/util'

```
And same goes for the Web UI on 127.0.0.1:3000
<img width="586" alt="Screen Shot 2021-03-03 at 18 20 18" src="https://user-images.githubusercontent.com/406826/109836770-31841d80-7c4d-11eb-90f0-bd849d3c98a7.png">

I've already checked locales/en and there's no messages.po in it (There is message.po for Spanish).

--- a/docker-compose.development.yml
+++ b/docker-compose.development.yml
@@ -58,6 +58,8 @@ services:
             curl -sL https://deb.nodesource.com/setup_14.x | bash -
             apt-get install -y nodejs
             npm install
+            npm run extract
+            npm run compile
             npm start --loglevel verbose
     user: root
     stdin_open: true

